hypha/apply/projects/urls.py

```diff
@@ -168,7 +168,7 @@
                         ),
                     ),
                 ],
-                "reports",
+                "vendor",
```

Comment:
As it is not getting used anywhere, I think we can just remove this whole path.
I looked into the commit history as well and it is added [here when forms were getting arranged](https://github.com/HyphaApp/hypha/commit/c2a3598ab98bbc741a6319ca26e41ac8e88611d7#diff-d63538e8198ab657b1878c6f6f4b998bf30824d6ad414a594131e740f7ee34ceR61) to multiple files. It doesn't make any sense in that commit as well, It might be the preparation to add vendor urls but later those urls were [added separately](https://github.com/HyphaApp/hypha/commit/9a2d4840e3dfdb8c432cf012acc5f761e70d5095#diff-d63538e8198ab657b1878c6f6f4b998bf30824d6ad414a594131e740f7ee34ceL40).